dev-guide/host-port-registry.md

```diff
@@ -142,6 +144,7 @@ Ports are assumed to be used on all nodes in all clusters unless otherwise speci
 | 10302 | various CSI drivers | storage | 4.7 | healthz |
 | 10303 | various CSI drivers | storage | 4.9 | healthz |
 | 11443 | kube-scheduler | workloads || recovery-controller |
+| 20000  | haproxy | sdn | 4.7 | on-prem internal loadbalancer, stats port |
```

Comment:
while I guess you _can_ use that port, for the others we tried to keep the convention that the localhost-only port was 20000 more than the externally-accessible port...

Reply:
That works for me, I wasn't aware of the convention before now. I've updated this and the runtimecfg patch accordingly.